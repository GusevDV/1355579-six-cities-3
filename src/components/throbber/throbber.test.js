import React from "react";
import renderer from "react-test-renderer";
import Throbber from "./throbber.jsx";

it(`Should Throbber component render correctly`, () => {

  const render = renderer.create(
      <Throbber/>
  )
  .toJSON();

  expect(render).toMatchSnapshot();
});
