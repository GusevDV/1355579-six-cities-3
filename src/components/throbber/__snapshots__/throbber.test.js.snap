// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Throbber component render correctly 1`] = `
<img
  alt="throbber"
  className="throbber"
  height={100}
  src="/img/throbber.svg"
  width={100}
/>
`;
